docs/reference/commandline/node_activate.md

```diff
@@ -0,0 +1,27 @@
+<!--[metadata]>
++++
+title = "node activate"
+description = "The node activate command description and usage"
+keywords = ["node, activate"]
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+# node activate
+
+    Usage:  docker node activate NODE [NODE...]
+
+    Activate a node in the swarm
+
+Activate a node into the swarm. This command targets a docker engine that is a manager in the swarm cluster.
+
+
+```bash
+$ docker node activate <node name>
```

Comment:
Or node ID, right? Is there a concise way to show that either works? Maybe another line with

```bash
docker node activate <node ID>
```